go.mod

```diff
@@ -20,7 +20,7 @@ require (
 	github.com/google/uuid v1.1.1 // indirect
 	github.com/inconshreveable/mousetrap v1.0.0 // indirect
 	github.com/influxdata/line-protocol v0.0.0-20180522152040-32c6aa80de5e
-	github.com/influxdata/pkg-config v0.2.0
+	github.com/influxdata/pkg-config v0.2.1-0.20200608212046-10c7c9815e64
```

Comment:
Did you tag a new release?

Reply:
If a new `pkg-config` update was required does this mean we need to tell people to update their local `pkg-config` install? Or will the change only be relevant to cross-compilation?

Reply:
It will only be relevant to cross compilation for this one but this does underscore the need to separate the build itself from the rest of the stuff `pkg-config` does at some point so that no updates to the tool are needed.

Reply:
Yeah, this is weird. I _did_ tag a new release, and even set it via `go get github.com/influxdata/pkg-config@0.2.2` - it pulled in the right version, but it's doing it via a git rev instead of the tag? I'll investigate.

Reply:
> Or will the change only be relevant to cross-compilation?

The change would only be relevant to cross-compilation for macOS targets, and in most places where we do releases (I assume the only regular time that we cross compile to macOS), it gets installed into the environment at the time of cutting the release. The make target I created verifies that.

Reply:
> Did you tag a new release?

Yeah, I'm not sure what's going on, because I did indeed `go get` via the tag, and it's still saying 0.2.1.